openmdao/core/problem.py

```diff
@@ -1443,6 +1444,8 @@ def check_totals(self, of=None, wrt=None, out_stream=_DEFAULT_OUT_STREAM, compac
         step_calc : string
             Step type for finite difference, can be 'abs' for absolute', or 'rel' for relative.
             Default is 'abs'.
+        show_progress : bool
+            Bool to show progress of check_totals
```

Comment:
I find it better to use less computer-science-like language:  "Set to True to show...